src/Umbraco.Web.UI.Client/src/views/propertyeditors/contentpicker/contentpicker.controller.js

```diff
@@ -413,16 +413,26 @@ function contentPickerController($scope, $q, $routeParams, $location, entityReso
             var missingIds = _.difference(valueIds, renderModelIds);
 
             if (missingIds.length > 0) {
-                return entityResource.getByIds(missingIds, entityType).then(function (data) {
 
+                var requests = [
+                    entityResource.getByIds(missingIds, entityType),
+                    entityResource.getUrlsByUdis(missingIds)
```

Comment:
Is there a way to configure the CMS such that missingIds here is an array of integers or is this just legacy hangover?